receiver/otlpreceiver/testdata/uds.yaml

```diff
@@ -1,7 +1,7 @@
 # The following entry demonstrates how to specify a Unix Domain Socket for the server.
 protocols:
   grpc:
-    transport: unix
+    transport_type: unix
```

Comment:
I think we should revert this back using the technique I mentioned on a comment above

Reply:
Will do

Reply:
Done